src/snowflake/cli/_plugins/spcs/services/commands.py

```diff
@@ -221,20 +225,68 @@ def logs(
     num_lines: int = typer.Option(
         500, "--num-lines", help="Number of lines to retrieve."
     ),
+    previous_logs: bool = typer.Option(
```

Comment:
why are we exposing previous_logs

Reply:
But as you mentioned, it's okay to keep the code this way. The previous_logs parameter is included in the system function, and if we remove it, it would be confusing to pass a false in every call. This is because we should only pass a value for since when previous_logs has a value. When calling the system function, we can also demonstrate the functionality of previous_logs using SnowCLI during the presentation.

Reply:
From end user perspective "Retrieve previous logs" is very ambiguous. How many previous logs?

Reply:
This is another feature not related to log streaming, it is the log from last terminated instance.